engine/rest-client/src/main/java/org/datacleaner/restclient/Serializator.java

```diff
@@ -59,6 +64,20 @@
         myModule.addSerializer(new MyEnumSerializer());
         objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
         objectMapper.registerModule(myModule);
+
+        //For better deserialize of enumeration in ComponentScope annotation. - We will ignore unknown values.
+        objectMapper.registerModule(new SimpleModule(){
+            @Override
+            public void setupModule(final SetupContext context) {
+                context.setMixInAnnotations(ComponentScope.ServiceType.class, ComponentScopeServiceTypeMixin.class);
+            }
+        });
+        objectMapper.registerModule(new SimpleModule(){
+            @Override
+            public void setupModule(final SetupContext context) {
+                context.setMixInAnnotations(ComponentScope.EntityType.class, ComponentScopeEntityTypeMixin.class);
+            }
+        });
```

Comment:
I think you should be able to do this using a single module with two calls to `context.setMixInAnnotations`. Not sure about that, and not really an issue, just an alternative :)